Plugins~/Src/MeshSyncClientBlender/msblenBinder.cpp

```diff
@@ -635,6 +635,23 @@ std::string abspath(const std::string& path)
     }
 }
 
+std::string getBlenderVersion()
+{
+    try {
+        auto global = py::dict();
+        auto local = py::dict();
+        py::eval<py::eval_mode::eval_statements>(
+            "import bpy\n"
+            "ret = bpy.app.version_string"
+            , global, local);
+        return (py::str)local["ret"];
+    }
+    catch (py::error_already_set& e) {
+        muLogError("%s\n", e.what());
+        return "";
+    }
```

Comment:
There is a C API to get the version string https://github.com/blender/blender/blob/master/source/blender/blenkernel/BKE_blender_version.h  , have you tried using it?

Reply:
That would give us the version the plugin was compiled with. I want to get the version from the blender version currently running which may be slightly different.

Reply:
Oh right, my bad